### ISSUE TITLE
[Masthead]: A11y - The table summary duplicates the caption

### ISSUE THREAD

**Issue by rohitrajmishra** [carbon-design-system/carbon-for-ibm-dotcom#8346]
### Description

===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: 1.3.1 Info and Relationships
IBM Guidance: https://www.ibm.com/able/requirements/requirements/?version=7_1#1_3_1
Test Tool Type: IBM Equal Access Accessibility Checker

===== DEFECT DESCRIPTION =====
The table summary must not duplicate the caption. Duplicating a table’s caption and its summary description is repetitive and slows productivity for Assistive Technology users. If used, the summary should provide complementary information not already available in the caption.

=====STEPS TO REPRODUCE =====

1. Open https://dev.console.test.cloud.ibm.com
2. Login with IBM ID & password
3. Click Help ->Docs
4. Click All product docs
5. Click Discovery
6. Click Connecting to your data ->Creating collections
7. Check Accessibility

===== EXPECTED RESULT =====
- If the table caption describes the table purpose and structure fully, remove the summary;

- Or, if information beyond the caption is needed, write a summary that describes the table structure without duplicating information in the caption. The summary should be referenced with an aria-describedby attribute. The summary attribute is obsolete in HTML5.

<img width="1430" alt="image" src="https://media.github.ibm.com/user/140782/files/2c2bbc00-4df6-11ec-9871-e2a882fb0706">

### Component(s) impacted

Masthead

### Browser

Chrome, Safari, Firefox, Microsoft Edge

### Carbon for IBM.com version

vv1.11.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

cloud.ibm.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://codesandbox.io/s/github/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/react/examples/codesandbox/components/Masthead?file=/src/App.js

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by annawen1** [comment]
I ran this on our latest verison: https://www.ibm.com/standards/carbon/web-components/iframe.html?id=components-masthead--default and did not see this error appear.

<img width="1253" alt="Screen Shot 2022-03-15 at 1 58 55 PM" src="https://user-images.githubusercontent.com/54281166/158441963-263a9b70-cf66-473b-9202-7cb3e562acab.png">

Closing this ticket
